scss/templates/user.scss

```diff
@@ -333,3 +315,102 @@
     }
   }
 }
+// User Navigation
+.user-secondary-navigation {
+  .new-private-message {
+    @include user-profile-btn;
+    width: 9rem;
+    margin-bottom: 1rem;
+  }
+
+  .nav-stacked {
+    li {
+      margin-bottom: 1.4375rem;
+
+      a {
+        color: $gray-100;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 18px;
```

Comment:
for `font-size`, should we use px or rem?

Reply:
rem, my bad

Reply:
Fixed, no more `px` 💯